solutions/ocp-ai/main.tf

```diff
@@ -0,0 +1,49 @@
+# Retrieve existing OpenShift cluster details
+data "ibm_container_vpc_cluster" "cluster" {
+  name = var.cluster_name
+}
+# # Retrieve existing  OpenShift cluster version details
+data "ibm_container_cluster_versions" "cluster_versions" {
+  resource_group_id = data.ibm_container_vpc_cluster.cluster.resource_group_id
+}
+# Retrieve existing  existing details
+data "ibm_container_addons" "addons" {
```

Comment:
If the idea here is to get the existing addons information from the cluster, use a flag to get details only when an existing cluster is provided.